### DISCUSSION TITLE
Get Token

### DISCUSSION THREAD

**Discussion by ThomasKovala** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#681] [Q&A] [answered]
<img width="1527" alt="Screen Shot 2022-03-20 at 21 47 10" src="https://user-images.githubusercontent.com/101996296/159185354-e158fb82-0c68-4dd5-ac27-afc55086fbb1.png">
Hi, on Mac when I get to the last step where token should appear I see just localhost in the url and nothing else show up.

**Comment by im-coder-lg** [comment]
Hmm that's not supposed to happen, can you try using other browsers?

**Comment by im-coder-lg** [comment, marked as answer]
I think you have this issue: https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues/342#issuecomment-1072913967

Which was resolved by both comments here: https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues/671#issuecomment-1072883616 https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues/671#issuecomment-1072699291
Hope I positioned the links properly.

**Reply by ThomasKovala** [reply]
Thanks, it worked.
